scripts/run_pipeline

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# This script is used to run the pipeline of:
+# 1. Creating the prioritized list of climate actions
+# 2. Generating the enriched JSON file for the frontend
+# 3. Uploading the files to the AWS S3 bucket
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+set -o pipefail
+
+# Paths
+VENV_PYTHON="../.cap/Scripts/python.exe"
```

Comment:
This will be different on MacOS and Linux, so let's make sure we use the correct executable for each platform (e.g. by finding the file in the directory or checking which OS we're running on and choosing the right file name).

Reply:
I added a check for the OS and choose the path dynamically according to the OS. Thanks